rsts/concepts/data_management.rst

```diff
@@ -15,35 +34,38 @@ Let us consider a simple Python task:
    def my_task(m: int, n: str, o: FlyteFile) -> pd.DataFrame:
       ...
 
-In the above code sample, ``m, n, o`` are inputs to the task.
+In the above code sample, ``m``, ``n``, ``o`` are inputs to the task.
 ``m`` of type ``int`` and ``n`` of type ``str`` are simple primitive types, while ``o`` is an arbitrarily sized file.
-All of them from Flyte's point of view are ``data``.
+All of them from Flyte's point of view, are ``data``.
 The difference lies in how Flyte stores and passes each of these data items.
 
 For every task that receives input, Flyte sends an **Inputs Metadata** object, which contains all the primitive or simple scalar values inlined, but in the case of
-complex, large objects, they are offloaded and the `Metadata` simply stores a reference to the object. In our example, ``m, n`` are inlined while
-``o`` and the output ``pd.DataFrame`` are offloaded to an object store, and reference is captured in the metadata.
+complex, large objects, they are offloaded and the `Metadata` simply stores a reference to the object. In our example, ``m``, and ``n`` are inlined while
+``o`` and the output ``pd.DataFrame`` are offloaded to an object store, and their reference is captured in the metadata.
 
-`Flytekit TypeTransformers` make it possible to use complex objects as if they are available locally - just like persistent filehandles. But Flyte backend only deals with
+``Flytekit TypeTransformers`` make it possible to use complex objects as if they are available locally - just like persistent filehandles. But Flyte backend only deals with
```

Comment:
```suggestion
`Flytekit TypeTransformers` make it possible to use complex objects as if they are available locally - just like persistent filehandles. But Flyte backend only deals with
```
That's not inline code, so let's have single backticks to print italicized text.